tests/backends/jax/gpr_jax.py

```diff
@@ -0,0 +1,83 @@
+import jax
+import jax.numpy as jnp
+import jax.random as jr
+import jax.scipy as jsp
+import meshzoo
+import numpy as np
+import polyscope as ps
+
+import geometric_kernels.jax  # noqa
```

Comment:
There's an [__init__.py](https://github.com/GPflow/GeometricKernels/blob/main/tests/__init__.py) file in the tests which loads all the backends. It would be good to have jax in there as well.
```suggestion
```